File: Day7/Solution.c
int main(){
    int n;
    scanf("%d",&n);
    int *arr = malloc(sizeof(int) * n);
    for(int arr_i = 0; arr_i < n; arr_i++){
       scanf("%d",&arr[arr_i]);
    }
    for(int i=n-1;i >= 0 ; i--){
            printf("%d ",arr[i]);
        }
    return 0;
}

File: Day7/Solution.go
package main
import (
  "fmt"
)

func main() {
  var size int;
  fmt.Scanf("%d", &size)
  
  var array = make([]int, size)
  for i := 0; i < size; i++ {
    fmt.Scanf("%d", &array[i])
  }
  
  for i := size; i > 0; i-- {
    fmt.Printf("%d ", array[i-1])
  }
}
